assets/css/_properties_panel.scss

```diff
@@ -18,6 +18,13 @@
       width: 100%;
     }
   }
+  @media screen and (max-width: $mobile-max-width) {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    width: 100%;
```

Comment:
Other comments from above apply here, but also note that the `width: 100%` here is redundant with `width: 100vw` in the block above.